src/applications/travel-pay/containers/pages/ClaimStatusExplainerPage.jsx

```diff
@@ -0,0 +1,213 @@
+    return (
+      <div className="vads-l-grid-container vads-u-padding-y--3">
+        <va-loading-indicator
+          label="Loading"
+          message="Please wait while we load the application for you."
+          data-testid="travel-pay-loading-indicator"
+        />
+      </div>
+    );
+  }
+
+  if (!appEnabled) {
+    window.location.replace('/');
+    return null;
+  }
+
+  return (
+    <Element name="topScrollElement">
+      <article className="usa-grid-full vads-u-padding-bottom--0">
+        <BreadCrumbs />
+        <h1 tabIndex="-1" data-testid="status-explainer-header">
+          What does my claim status mean?
+        </h1>
+        <div className="vads-l-row vads-u-margin-x--neg2p5">
+          <div className="vads-l-col--10 vads-u-padding-x--2p5 ">
+            <p>
+              VA uses many statuses to track claims from before you submit to
+              after you receive payment. This page provides descriptions for
+              what each status means. If you continue to have questions about
+              your claim, please contact your local VA Medical Center (VAMC) and
+              ask to speak with the Beneficiary Travel department.
+            </p>
+            <p>
+              On the Dashboard and Claims pages, each claim has a status. These
+              pages also include filters which allow you to filter by one of the
+              following categories: Saved or Incomplete, Under VA Review, and
+              Closed. The claim categories and statuses within Beneficiary
+              Travel Self Service System (BTSSS) are:
+            </p>
+
+            <h2 className="vads-u-font-size--h3">Saved or Incomplete</h2>
+            <p>
+              These are claims that the Beneficiary Travel office cannot
+              process. Either you have not submitted the claim, or you submitted
+              a claim without required information. This category of claim
+              status includes:
+            </p>
+            <ul>
+              <li>
+                <b>Incomplete — </b>
+                You submitted a claim without required expense information. You
+                must provide the required information for BTSSS to process the
+                claim.
+              </li>
+              <li>
+                <b>Saved — </b>
+                You saved changes to your claim, but you did not submit it to
+                BTSSS for review. Submit the claim so BTSSS can begin processing
+                your claim.
+              </li>
+            </ul>
+
+            <h2 className="vads-u-font-size--h3">Under VA Review</h2>
+            <p>
+              {' '}
+              These claims require action from VA. If VA needs more information
+              from you, your Travel Clerk will contact you.
+            </p>
+            <ul>
+              <li>
+                <b>In Process — </b>
+                You submitted a claim, and now BTSSS is reviewing your claim.
```

Comment:
A pattern I've seen before is to have these statuses in a data structure like:
```javascript
const STATUSES = {
  name: 'In Process',
  description: 'blah'
}
```

What are your thoughts on doing this? I can see a case in the future where we may want to quickly display a single status or a small collection of statuses, and having them in a data structure would make that way less redundant. 

However, I don't want to early optimize. Thoughts?

Reply:
Yeah, I debated trying to figure out some way to have not such a huge block of text on that page. How do you feel about adding like a `constants` file or something, then all statuses and definitions are in one place to pull in wherever/however we need?

So, something like 

```
const STATUSES = [
  {
    type: 'Saved or Incomplete',
    statuses: [
      {
        name: 'Incomplete',
        description: 'blah',
      },
      {
        name: 'Saved',
        description: 'blah',
      },
    ],
  },
];
```

Reply:
Something like that would fit the bill imo

Reply:
✅